accessibility/common-remediations.md

```diff
@@ -0,0 +1,200 @@
+# Common remediations
+
+This page describes some common accessibility issues, and gives examples and suggestions for repairs. 
+
```

Comment:
Can we have a table of contents too? Will make it easier for us to see if the order of the sections makes sense 😬

Reply:
yes actually I was going to add it once we have most of the points covered